tests/Microsoft.Identity.Test.Unit.net45/RequestsTests/OboFlowTests.cs

```diff
@@ -1,932 +0,0 @@
-﻿/*
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using System.Net.Http;
-using System.Threading.Tasks;
-using Microsoft.Identity.Client;
-using Microsoft.Identity.Client.Internal.Cache;
-using Microsoft.Identity.Client.Internal.Http;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Test.MSAL.NET.Unit.Mocks;
-
-namespace Test.MSAL.NET.Unit.RequestsTests
-{
-    class OboFlowTests
```

Comment:
These are just some ADAL tests for OBO that were never really properly moved. They are useless now.